masq/src/non_interactive_mode.rs

```diff
@@ -518,4 +503,95 @@ mod tests {
             ])
         )
     }
+
+    #[derive(Debug)]
+    struct FakeCommand {
+        output: String,
```

Comment:
Is this field ever used? (It's set, sure enough, but is its value ever read?)

Reply:
Oh no, seems needless, no red lights when made removed.